src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -97,6 +85,7 @@ public void afterPropertiesSet() {
     fafService.addOnMessageListener(GameLaunchMessage.class, this::updateLobbyModeFromGameInfo);
     fafService.addOnMessageListener(DisconnectFromPeerMessage.class, message -> iceAdapterProxy.disconnectFromPeer(message.getUid()));
     fafService.addOnMessageListener(IceServerMessage.class, message -> iceAdapterProxy.iceMsg(message.getSender(), message.getRecord()));
+    iceAdapterProxy = newIceAdapterProxy();
```

Comment:
depends on what the method depends on either in constructor or if possible in declaration